src/mpi/romio/adio/ad_gpfs/ad_gpfs_aggrs.c

```diff
@@ -641,8 +645,18 @@ void ADIOI_GPFS_Calc_others_req(ADIO_File fd, int count_my_req_procs,
     int i;
     ADIOI_Access *others_req;
 
+#if MPI_VERSION >= 1234 || (MPI_VERSION == 1234 && MPI_SUBVERSION > 1234)
```

Comment:
What is `1234`? `>= 4` should work, right?

Reply:
I had just put that as a placeholder, but I guess MPI-4 is getting close enough to official to say >= 4

Reply:
MPI-4 is fully implemented in the main branch.